src/Ombi.Schedule/Jobs/Emby/EmbyPlayedSync.cs

```diff
@@ -98,13 +113,117 @@ private async Task ProcessMovie(EmbyMovie movieInfo, OmbiUser user, ICollection<
                 UserId = user.Id
             };
             // Check if it exists
-            var existingMovie = await _repo.Get(userPlayedMovie.TheMovieDbId, userPlayedMovie.UserId);
+            var existingMovie = await _movieRepo.Get(userPlayedMovie.TheMovieDbId, userPlayedMovie.UserId);
             var alreadyGoingToAdd = content.Any(x => x.TheMovieDbId == userPlayedMovie.TheMovieDbId && x.UserId == userPlayedMovie.UserId);
             if (existingMovie == null && !alreadyGoingToAdd)
             {
                 content.Add(userPlayedMovie);
             }
         }
+
+        private async Task ProcessTvUser(EmbyServers server, OmbiUser user, string parentId = default)
+        {
+            EmbyItemContainer<EmbyEpisodes> episodes;
+            if (recentlyAdded)
+            {
+                var recentlyAddedAmountToTake = 10; // to be adjusted?
+                episodes = await Api.GetTvPlayed(server.ApiKey, parentId, 0, recentlyAddedAmountToTake, user.ProviderUserId, server.FullUri);
+                // Setting this so we don't attempt to grab more than we need
+                if (episodes.TotalRecordCount > recentlyAddedAmountToTake)
```

Comment:
```suggestion
                if (episodes.TotalRecordCount >= recentlyAddedAmountToTake)
```

Reply:
If the two variables are equal, it's not necessary the assign one to the other is it?
For the record, this is a copy and paste from the episode sync (from https://github.com/sephrat/Ombi/blob/9fe7ec115e8657ed77028cbdfce1e449bfb22c1f/src/Ombi.Schedule/Jobs/Emby/EmbyEpisodeSync.cs#L112).